### DISCUSSION TITLE
UI/UX development contributions.

### DISCUSSION THREAD

**Discussion by HowdyHolto** [FundamentalFrequency/.github#10] [General]
@stonepreston I tried looking through the files last night, but my gut is you're currently doing everything in code. I'd love to contribute with my product design background but I'm likely only able to work within graphic formats to begin with... I can of course submit redlines/color codes, etc when done, but is there any way to get hold of the current UI flows/visual formatting? I can't run the emulator just yet as I'm on macOS. I will be installing a virtual machine soon, but haven't had the chance yet. 

I would also be curious if I'm outputting files, what some of the open source or free tools can accept if folks have experience there. I'm usually working in some combination of Adobe, Sketch or Affinity suite. I can of course just output everything to SVG if that works best for the backend dev.

Any advice from the community is welcome here. First time contributing on an open source project. Lots to learn!

**Comment by stonepreston** [comment]
Yes all the UI is in code only, JUCE does not have a layout editor or anything like that. 

If you would like to post graphical elements/ideas, it would be best to provide SVGs if possible, and to use FOSS programs like Inkscape/gimp for manipulation 

regarding the emulator, you should be able to build both the DAW and the emulator from source on OSX. The JUCE tutorials should provide instructions on how to get things setup on OSX. I can’t confirm 100 percent it will work as I’ve never tried, but it would be worth a shot and should in theory work.

**Reply by HowdyHolto** [reply]
I figured that might be the case. This is helpful. I used Inkscape eons ago. I'll take a look. I'm thinking I can just bring files over to it and save them out in that format for folks. 

To keep things tidy, is there a specific place you want us to post UI/UX shots? Apologies that I'm so shite at GitHub. You would think with all the devs I've worked with over the years... hahaha.

**Reply by stonepreston** [reply]
@HowdyHolto Ive added a UI/UX label, so I would just post your ideas to the ideas category and label it as UI/UX. Also TBH ive never used GitHub discussions before either, so its a learning experience for me as well.

**Reply by HowdyHolto** [reply]
Oh perfect. Yeah, as someone with ADHD I don't know how you're navigating all the input :-), but this is impressive with how well it's been received by the community. Very inspiring. Thanks for adding the label. As soon as I can get the emulator running in some way I'll contribute there for sure.

**Reply by stonepreston** [reply]
I went ahead and added a UI/UX category as well in addition to the label. Didnt see that I could add more categories so ive added categories for UI/UX, software, and CAD related stuff.
